### PR TITLE
Fix issue #291 - broken compatibility with Keras since 2.2.1 due to wrong import

### PR DESCRIPTION
**Pull request by wilderrodrigues**
Although there is another PR with this fix, it seems that it contains test errors and I also observed that the import is not right, as it is not importing from the Keras backend, as seen here:

```
https://github.com/keras-team/keras/blob/d40656e5799dfd22e96b8bab217638b2934a6894/tests/keras/utils/conv_utils_test.py

K.normalize_data_format('channels_middle')
```

This has been tested on a Databricks cluster and is working.

image

### CODE REVIEW

**Comment by keisen** [comment]
Hi, @wilderrodrigues 

This PR is essentially as same as #292.
`keras.backend.normalize_data_format` is an alias for `keras.backend.common.normalize_data_format` in currently Keras.

https://github.com/keras-team/keras/blob/4db04d8701a64ee249dfe6ce09ebd89063bfb99c/keras/backend/tensorflow_backend.py#L22

It is confusing if there are multiple PRs that do the same fixes.

Thanks.

**Comment by wilderrodrigues** [comment]
Hi @keisen,

Indeed, you are right.

I will close this PR and try to fix the issues with the tests on your instead.

The test could be failing to due a latest Numpy release. I compared the latest PR with green tests and mine/yours to see the difference. There are lots of complains coming from Numpy.

Cheers.
